src/containers/signup/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
```

Comment:
I would prefer to get rid of this, but I get if we just have it and never use it